packages/scxa-marker-gene-heatmap/src/MarkerGeneHeatmap.js

```diff
@@ -149,9 +145,9 @@ const MarkerGeneHeatmap = (props) => {
       opposite: true,
       categories: xAxisCategories,
       labels: {
-        useHTML: true,
+        useHtml: true,
```

Comment:
[The latest version of the Highcharts API has changed this attribute to be `useHTML`](https://api.highcharts.com/highcharts/xAxis.labels.useHTML).